docs/developer/decision-records/2022-06-20-mvd-onboarding/README.md

```diff
@@ -0,0 +1,28 @@
+# Participants onboarding in MVD dataspace
+
+This document outlines the infrastructure steps for the scenario of including participants in the data space.
+
+## Precondition - GAIA-X membership
+
+To join the MVD dataspace, participants must have Verifiable Credentials signed by GAIA-X Authority which prove GAIA-X membership.
+
+## Onboarding process - GAIA-X membership verification
+
+During the onboarding process the GAIA-X membership claim is being verified by the Dataspace Authority.
+The onboarding flow is presented in the [distributed authorisation sub-flow doc](https://github.com/agera-edc/MinimumViableDataspace/tree/feature/20-rs-adr-target/docs/developer/decision-records/2022-06-16-distributed-authorization).
```

Comment:
Link to the ADR in downstream repo will be changed to the edc repo once the upstream PR is merged.

Reply:
done